boost-common/src/main/java/io/openliberty/boost/common/utils/LibertyServerConfigGenerator.java

```diff
@@ -218,7 +218,11 @@ public void addBootstrapProperties(Properties properties) throws IOException{
     public void addConfigForApp(String artifactId, String version){
     	Element appCfg = doc.createElement(APPLICATION);
     	appCfg.setAttribute("context-root", "/");
-    	appCfg.setAttribute("location", artifactId+"-"+version+ "." + ConfigConstants.WAR_PKG_TYPE);
+        if(version == null) {
+            appCfg.setAttribute("location", artifactId + "." + ConfigConstants.WAR_PKG_TYPE);
+        } else {
+            appCfg.setAttribute("location", artifactId + "-" + version + "." + ConfigConstants.WAR_PKG_TYPE);
```

Comment:
Maybe a dumb question, but is a 'null' version a thing in Gradle?   It's not in Maven...     Maybe this is just defensive code...

Reply:
The war task can have a null version so when I was testing we were creating an application element with null as part of the app location.

Reply:
So this is the version of the war task specifically?  Not the artifact as a whole (as in Maven)?   Maybe this concept doesn't exist in Maven (of the artifact always having a GAV of its own?)  Sorry for my ignorance.

Reply:
Right, the project might not have a version assigned to it or the war, which is fine in gradle.